src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -117,5 +117,12 @@
      * Removes the currently displayed {@link Title} from the player's screen.
      */
     void clearTitle();
+    
+    /**
+     * Get a player's ping.
+     * 
+     * @return The player's ping
+     * /
```

Comment:
You need to close the Javadoc comment here: https://travis-ci.org/SpongePowered/SpongeAPI/builds/45701022#L217-L220
> /home/travis/build/SpongePowered/SpongeAPI/src/main/java/org/spongepowered/api/entity/playe/Player.java:121: unclosed comment